Makefile

```diff
@@ -0,0 +1,24 @@
+image_tor := patrickod/tor
+
+.PHONY: start stop run build tor
+
+start:
+	docker start tor
+
+stop:
+	docker stop tor
+
+run: tor
+
+tor:
+	-@docker rm -f "$@"
+	docker run -d \
+		--name "$@" \
+		-e "TZ=Europe/Berlin" \
```

Comment:
I'm guessing this is your locale at the moment but I think we should take this out before merging for folks?